src/main/java/core/basesyntax/io/CsvReader.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.io;
+
+import core.basesyntax.io.exception.CsvException;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.OperationType;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CsvReader {
+    private static final String DELIMITER = ",";
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
+    public List<Operation> readOperations(String fileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            return reader.lines()
+                    .map(this::csvStringToOperation)
+                    .collect(Collectors.toList());
+        } catch (IOException e) {
+            throw new CsvException("Cannot access file with name = [" + fileName + "]");
```

Comment:
Instead of throwing a new instance of `CsvException` directly, it's better to wrap the original `IOException` inside it to preserve the stack trace and provide more context for debugging. Consider re-throwing the exception like this: `throw new CsvException("Cannot read file: " + fileName, e);`